flytekit/remote/remote.py

```diff
@@ -2542,7 +2542,7 @@ def sync_node_execution(
             launched_exec = self.fetch_execution(
                 project=launched_exec_id.project, domain=launched_exec_id.domain, name=launched_exec_id.name
             )
-            self.sync_execution(launched_exec)
+            self.sync_execution(launched_exec, sync_nodes=True)
```

Comment:
nit
```suggestion
            self.sync_execution(launched_exec, sync_nodes=sync_nodes)
```

Reply:
so this is under the `sync_node_execution` function. There is no `sync_nodes` variable. sync_node_execution is only called when `sync_node=True`.

Let me know if it makes sense.

Reply:
ah, sorry, my bad